tournament/src/tournament/settings.py

```diff
@@ -75,8 +76,12 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
+    'django.middleware.common.CommonMiddleware',
 ]
 
+CORS_ALLOWED_ORIGINS = ['https://*']
+
```

Comment:
For allow all, please use :

CORS_ALLOW_ALL_ORIGINS = True
CORS_ALLOWED_ORIGINS = ['*']